lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -2122,6 +2122,10 @@ hasTotalCollateral (CardanoApi.Tx (CardanoApi.TxBody content) _) =
         CardanoApi.TxTotalCollateralNone -> False
         CardanoApi.TxTotalCollateral _ _ -> True
 
+-- | A type-constrained variant of 'it'.
+itIO :: String -> IO () -> Spec
```

Comment:
Perhaps we could add a TODO here and link it to a ticket that reminds us to remove this when possible?

Reply:
Fixed in 868dbcdb21085da7814b2c936a8d90cafa60e817.